pygsti/tools/matrixtools.py

```diff
@@ -682,11 +682,16 @@ def approximate_matrix_log(m, target_logm, target_weight=10.0, tol=1e-6):
     assert(_np.linalg.norm(m.imag) < 1e-8), "Argument `m` must be a *real* matrix!"
     mx_shape = m.shape
 
+    #
+    #   Riley note: I'd like to remove all commented-out code in this function.
+    #   @Corey or @Stefan -- you okay with that?
+    #
+
```

Comment:
If you're referring to that block on the bottom beginning with 'OLD DEBUG' then go for it.

Reply:
Nah, that's in a different function. I've actually come to to realize that this ``approximate_matrix_log`` function is doing something quite weird, so I think keeping comments for posterity's sake is reasonable.